vfs/vfs.go

```diff
@@ -13,9 +13,29 @@ import (
 	"time"
 
 	"github.com/cozy/cozy-stack/couchdb"
+	"github.com/cozy/cozy-stack/couchdb/mango"
 	"github.com/spf13/afero"
 )
 
+// Indexes is the list of required indexes by the VFS inside CouchDB.
+var Indexes = []mango.Index{
+	mango.Index{
+		Name:  "FileFromParent",
```

Comment:
@aenario Oh i see. Thanks for the info. I'll just leave a comment in our Go code then.